recipes/torchsort/build.bat

```diff
@@ -0,0 +1,2 @@
+set TORCH_CUDA_ARCH_LIST=5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX
```

Comment:
FYI, i've sometimes been lazy and defined these set/export stuff in the meta.yaml itself to avoid the need for a 2 line script.

Reply:
I wanted to try that too, but I couldn't find a way to do it portably via rattler-build. I suppose I could go for separate Unix/Windows line but probably not worth the effort at this point.